src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
Here you can use destructuring to improve code readability.

Usage example:

```suggestion
  const { name, age, sex, isMarried, partnerName } = person;
  const partner = person.sex === 'm' ? 'wife' : 'husband';
  
  return (
    <section className="Person">
      <h2 className="Person__name">{`My name is ${name}`}</h2>
```